lib/include/CL/SDK/Image.hpp

```diff
@@ -10,7 +10,7 @@ namespace cl {
 namespace sdk {
     struct Image
     {
-        int width = 0, height = 0, pixel_size = 1;
+        size_t width = 0, height = 0, pixel_size = 1;
```

Comment:
Given there are changes below to `reinterpret_cast` these back to `int` in one location below is this the correct change?  Should the other side of whatever was triggering the warning be changed?